x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/SecurityUsageTransportAction.java

```diff
@@ -183,15 +186,27 @@ protected void masterOperation(
         }
     }
 
+    private Map<String, Object> remoteClusterServerUsage() {
+        if (TcpTransport.isUntrustedRemoteClusterEnabled() && XPackSettings.SECURITY_ENABLED.get(settings)) {
+            // TODO: replace available with license check
+            return Map.of("available", true, "enabled", RemoteClusterPortSettings.REMOTE_CLUSTER_SERVER_ENABLED.get(settings));
```

Comment:
I merged https://github.com/elastic/elasticsearch/pull/94638 yesterday. We can already resolve this `TODO`:


```suggestion
            return Map.of(
                "available",
                Security.CONFIGURABLE_CROSS_CLUSTER_ACCESS_FEATURE.checkWithoutTracking(licenseState),
                "enabled",
                RemoteClusterPortSettings.REMOTE_CLUSTER_SERVER_ENABLED.get(settings)
            );
```

Reply:
Thanks for catching it. I forgot to update.